samples/sensor/adxl372/README.rst

```diff
@@ -0,0 +1,94 @@
+.. _adxl372:
+
+ADXL372: Three Axis High-g I2C/SPI Accelerometer
+################################################
+
+Description
+***********
+
+This sample application produces slightly different outputs based on the chosen
+driver configuration mode. In Measuring Mode with trigger support,
+the acceleration on all three axis is printed in m/s^2 at the sampling rate (ODR).
+In polled Measuring Mode the instantaneous acceleration is polled every 2 seconds.
+In most high-g applications, a single (3-axis) acceleration sample at the peak
+of an impact event contains sufficient information about the event, and the
+full acceleration history is not required.
+In this Max Peak Detect Mode the device returns only the over threshold
+peak acceleration between two consecutive sample fetches or trigger events.
+Instead of printing the acceleration on all three axis, the sample application
+calculates the vector magnitude (root sum squared) and displays the result
+in g rather than in m/s^2, together with an bar graph.
+
+
+References
+**********
+
+ - ADXL372: http://www.analog.com/adxl372
+
+Wiring
+*******
+
+This sample uses the ADXL372 sensor controlled either using the I2C or SPI interface.
+Connect supply **VDD**, **VS** and **GND**. The supply voltage can be in
+the 1.6V to 3.5V range.
+
+I2C mode
+========
+
+Connect Interface: **SDA**, **SCL** and optionally connect the **INT1** to a
+interrupt capable GPIO. It is a requirement that **SCLK** must be connected to
+**GND** in I2C mode. Depending on the baseboard used, the **SDA** and **SCL**
+lines require Pull-Up resistors.
+Note: When sharing an SDA bus, the ADXL372 Silcon REV < 3  may prevent
```

Comment:
If this is an important note, you can write this as:

    .. note::

       When sharing an SDA bus, the ADXL372 Silcon REV < 3  may prevent
       communication with other devices on that bus.

Reply:
ok great